nms/app/fbcnms-projects/magmalte/app/views/network/NetworkDashboard.js

```diff
@@ -86,8 +88,8 @@ export default function NetworkDashboard() {
       </div>
 
       <AppBar position="static" color="default" className={classes.tabBar}>
-        <Grid container>
-          <Grid item xs={6}>
+        <Grid container direction="row" justify="flex-end" alignItems="center">
```

Comment:
isn't direction="row" default ? do we have to add this in here?

Reply:
Yeah, that's redundant. I'll remove it.